src/application/components/Enum.js

```diff
@@ -0,0 +1,41 @@
+// @flow
```

Comment:
We could still seperate them if we still used the same component? We might want to put them in different places but I can't imagine the way they look we would change? Seems like a bit of duplication with the css, if you think it would make the Model.js file too complex then we can leave as is.